packages/ListBoxWithTags/src/ListBoxWithTags.js

```diff
@@ -74,6 +74,8 @@ const renderTrigger = ({ t, size, selectedOptions, onRemove, renderTag, tagLabel
 
   return (
     <sc.Trigger
+      aria-haspopup="true"
```

Comment:
just `aria-haspopup`. No need to assign to "true"

Reply:
Actually no because `aria-haspopup` expects a string of "true" no the actual value of true

Reply:
Not sure what you mean, both ways ends up adding `aria-haspopup="true"` to the dom

Reply:
Hm, i wonder why we use `aria-haspopup="true"` in 3 other places then

Reply:
Perhaps update the 3 other places as well if it's not necessary? it's a small cleanup anyway